build.gradle

```diff
@@ -566,6 +566,23 @@ project(':iceberg-nessie') {
   }
 }
 
+project(':iceberg-dell') {
+  dependencies {
+    implementation project(':iceberg-core')
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation 'com.emc.ecs:object-client-bundle'
+
+    testImplementation "org.slf4j:jcl-over-slf4j"
```

Comment:
why is this needed? I don't see logging used in test.

Reply:
Removed.